server/protocols/protocols-smtp/src/main/java/org/apache/james/smtpserver/JamesRcptCmdHandler.java

```diff
@@ -43,7 +43,7 @@ public final void setDomainList(DomainList domainList) {
     @Override
     public String getDefaultDomain() {

```

Comment:
Shouldn't this return a domain?

Reply:
It could have but there's no real gain, we are very close to the protocol layer where we need to output strings anyway